_posts/en/newsletters/2021-02-03-newsletter.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Bitcoin Optech Newsletter #134'
+permalink: /en/newsletters/2021/02/03/
+name: 2021-02-03-newsletter
+slug: 2021-02-03-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a blog post about how a small change to
+the Script language after taproot activation could enable increased
+contract flexibility and includes our regular sections with brief
+descriptions of releases, release candidates, and notable changes to
+popular Bitcoin infrastructure projects.
+
+## News
+
+- **Replicating `OP_CHECKSIGFROMSTACK` with BIP340 and `OP_CAT`:** Andrew
+  Poelstra authored a [blog post][poelstra 340cat] about implementing
+  the functionality of ElementsProject.org's
```

Comment:
maybe link to https://elementsproject.org/ or just Elements or Elements Project (same for the topics page)

Reply:
> same for the topics page

For the topics page, this is not possible without a rewrite due to a limitation of Jekyll that the first paragraph can't contain any links because it's used as HTML metadata and may be included verbatim on other pages where the link can't be resolved.